cockpit-podman.spec.in

```diff
@@ -1,3 +1,20 @@
+#
+# Copyright (C) 2017-2020 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+#
```

Comment:
Spec files do not use or need copyright headers like regular code does, please do not include this.

Reply:
Meh, apparently @martinpitt was good with this for cockpit itself, so whatever...

Reply:
I understand that OpenSUSE requires them? And for Fedora/RHEL, it's "harmless", so I'm ok with it.